README.md

```diff
@@ -35,25 +35,38 @@ Sugarcoat was created to enable developers to produce rich UI documentation easi
 
 # Features #
```

Comment:
Let's link all feature bullet numbers with their options argument or section if there is one.